addons/stock/models/stock_rule.py

```diff
@@ -544,6 +544,8 @@ def run_scheduler(self, use_new_cursor=False, company_id=False):
                 self = self.with_env(self.env(cr=cr))  # TDE FIXME
 
             self._run_scheduler_tasks(use_new_cursor=use_new_cursor, company_id=company_id)
+        except Exception:
+            _logger.error("Error during stock scheduler", exc_info=True)
```

Comment:
You have to re-raise the exception :wink:

Reply:
`raise` or `raise e`

Reply:
why I need to re-raise ? If I re-raise, the finally will be never executed, no ?

**Before**
try
if Ok --> finally
if error --> finally

**After**
try:
if Ok --> finally
if error --> except (just log error)--> finally

Reply:
> If I re-raise, the finally will be never executed, no ?

No, the `finally` clause is always executed, even when the `except` clause raises an exception.  However, I could not find a clear explanation in this case in the Python documentation.  But simply try this:
```python
try: 
    raise Exception("Booh") 
except Exception: 
    print("except") 
    raise 
finally: 
    print("finally")
```